content/api/e-commerce-solutions/product-page-widget/_index.html

```diff
@@ -2,17 +2,21 @@
 title: Product page widget
 layout: api
 notanapi: true
-weight: 5
+menu:
+  api:
+    parent: e-commerce-solutions
+    identifier: product-page-widget
+    title: Product page widget
+    url: /api/product-page-widget
+weight: 1
 ---
 
-<main id="main" class="maxw48r w100p mvl phl jobs">
 <section class="dev-docscontent__section">
   <p>The product page widget is used on product pages to inform how quickly we can deliver the product to the customer.</p>
   <p>Our e-commerce widgets are fully integrated with relevant APIs, and represent a simplified way of presenting our services on your webshop. Using our widgets, you ensure that the way you present Posten’s delivery options is in line with our best practice recommendations.</p>
   <p>
     Implementation guide is available with the configuration of the widget on mybring.
   </p>
   <p><a href="https://www.mybring.com/shipping-guide/widgets/ps">Configure your widget (requires you to log in)</a></p>
-  <img src="/api/product-page-widget/product-page-widget.png" alt="Product page widget image"/>
-</section>
-</main>
+  <img src="/api/e-commerce-solutions/product-page-widget/product-page-widget.png" class="maxw40r" alt="Product page widget image"/>
```

Comment:
```suggestion
  <img src="/api/e-commerce-solutions/product-page-widget/product-page-widget.png" class="maxw24" alt=""/>
```
Just setting the size closer to what the widget is, and the alt text can be blank as it’s mostly decorative.